src/include/common/container/lock_free_array.h

```diff
@@ -22,7 +22,7 @@
 
 namespace peloton {
 
-#define LOCK_FREE_ARRAY_MAX_SIZE 1024 * 1024
+static const size_t kLockfreeArrayInitSize = 256;
 
 // LOCK_FREE_ARRAY_TEMPLATE_ARGUMENTS
 #define LOCK_FREE_ARRAY_TEMPLATE_ARGUMENTS template <typename ValueType>
```

Comment:
https://github.com/cmu-db/peloton/pull/1168#discussion_r177144365 from old PR:
`#define LOCK_FREE_ARRAY_TEMPLATE_ARGUMENTS template <typename ValueType>`
(Not your changes) I personally would avoid definitions here, especially because the def-name is longer than the actual content. Every modern IDE is able refactor the types if something changes.